com.github.sixpounder.GameOfLife.json

```
@@ -0,0 +1,50 @@
+{
+    "app-id" : "com.github.sixpounder.GameOfLife",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "43",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "game-of-life",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "env" : {
+            "G_MESSAGES_DEBUG" : "none",
+            "RUST_BACKTRACE" : "1",
+            "RUST_LOG" : "game-of-life=debug,glib=debug"
+        }
+    },
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "game-of-life",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "tag": "v0.1.0",
```

This tag doesn't exist:
`fatal: ambiguous argument 'v0.1.0': unknown revision or path not in the working tree.`

```suggestion
                    "tag": "0.1.0",
```

---

I was reviewing the code per your instructions, deleted the tag intentionally in the meanwhile. I just recreated it, and cargo-sources.json is now in this repo. Thank you for your patience :)

---

retagging is bad.

---

Created v0.1.1